src/displays/easeljs/states/BaseState.js

```diff
@@ -63,16 +63,16 @@
 
 		// @deprecated method for adding assets dynamically to task
 		this.on('loading', function(assets)
-		{
-			if (this.addTasks)
 			{
-				if (DEBUG) console.warn('addTasks has been deprecated, use loading event instead: e.g., state.on(\'loading\', function(assets){})');
-				this.addTasks(assets);
-			}
-		}, priority)
+				if (this.addTasks)
+				{
+					if (DEBUG) console.warn('addTasks has been deprecated, use loading event instead: e.g., state.on(\'loading\', function(assets){})');
+					this.addTasks(assets);
+				}
+			}, priority)
 
-		// Handle when assets are preloaded
-		.on('loaded', function(assets)
+			// Handle when assets are preloaded
+			.on('loaded', function(assets)
```

Comment:
Thought so as well, compared with the src file.. doesn't seem like anything major changed but just wanted to make sure.